src/containers/AdminCampaignList.jsx

```diff
@@ -293,9 +328,46 @@ class AdminCampaignList extends React.Component {
         ) : null}
         <CreateCampaignFromTemplateDialog
           organizationId={organizationId}
-          open={this.state.createFromTemplateOpen}
+          open={createFromTemplateOpen}
           onClose={() => this.setState({ createFromTemplateOpen: false })}
+          onCreateTemplateCompleted={this.onCreateTemplateCompleted}
```

Comment:
happy to change to this.handleCompleteCreateTemplate, but wondering why it's preferred to have the prop name be onComplete rather than onCreateTemplateComplete? I like more verbose props as it's _possible_ whatever child component can complete multiple different types of actions

Reply:
Actually, I feel like this `this.handleCreateTemplateCompleted` is more accurate than this version `this.handleCompleteCreateTemplate`

The first implies we're calling some side effects after "create template (mutation) has been completed" whereas the second suggests we're taking some action to complete "create template".

Does that make sense? I know this is a nit, but if it's an unspoken convention I'd rather talk about it.
So actually I'd prefer, `onCreateTemplateCompleted=this.handleCreateTemplateCompleted`

Reply:
im good with all your thoughts here, thanks for walking me through those! also cc @hiemanshu if you have any additional thoughts cause i've generally been following your (virtual) lead here

Reply:
Most libraries we've been using, and the "unspoken" convention has been to have props be small `onComplete`, `onClose`, `onOpen`, etc. And the function that handles it being more verbose `handleClickCreateNewCampaign` or similar. It keeps things in line with the current way we've been doing things, and these were done using something tips from Material UI components using similar conventions.